package.json

```diff
@@ -66,7 +66,6 @@
     "node-fetch": "^1.7.3",
     "node-sass": "^4.10.0",
     "qooxdoo": "^6.0.0-alpha",
-    "qooxdoo-sdk": "^6.0.0-alpha-20181226-a3c2b0b",
```

Comment:
ok ... why?  Isn't it a duplication because of `qooxdoo`?

Reply:
oh wait!  Doh!  I get it :)

Reply:
No! qooxdoo is just qx.Server you need for the compiler.
qooxdoo-sdk is the whole sdk. Never tried if it's working without qooxdoo. It can't without qooxdoo-sdk.